lib/resources/TransferReversals.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+
+/**
+ * TransferReversals is a unique resource in that, upon instantiation,
+ * requires a transferId, and therefore each of its methods only
+ * require the reversalId argument.
+ *
+ * This streamlines the API specifically for the case of accessing reversals
+ * on a returned transfer object.
+ *
+ * E.g. transferObject.reversals.retrieve(reversalId)
+ * (As opposed to the also-supported stripe.Transfer.retrieveReversal(transferId,
```

Comment:
Also fixed `ApplicationFeeRefunds`, `CustomerCards.js`, ` CustomerSubscriptions`, and `RecipientCards`.